crates/router/src/connector/stripe/transformers.rs

```diff
@@ -1573,34 +1588,119 @@ impl<F, T>
 // }
 
 #[derive(Debug, Deserialize)]
-pub struct StripeWebhookDataObjectId {
-    pub id: String,
+pub struct WebhookEventDataResource {
+    pub object: serde_json::Value,
 }
 
 #[derive(Debug, Deserialize)]
-pub struct StripeWebhookDataId {
-    pub object: StripeWebhookDataObjectId,
+pub struct WebhookEventObjectResource {
+    pub data: WebhookEventDataResource,
 }
 
 #[derive(Debug, Deserialize)]
-pub struct StripeWebhookDataResource {
-    pub object: serde_json::Value,
+pub struct WebhookEvent {
+    #[serde(rename = "type")]
+    pub event_type: WebhookEventType,
+    #[serde(rename = "data")]
+    pub event_data: WebhookEventData,
 }
 
 #[derive(Debug, Deserialize)]
-pub struct StripeWebhookObjectResource {
-    pub data: StripeWebhookDataResource,
+pub struct WebhookEventData {
+    #[serde(rename = "object")]
+    pub event_object: WebhookEventObjectData,
```

Comment:
The `#[serde(rename = "...")]` annotation is typically used when the field has the same name as a Rust keyword (for example `type`, `trait`, etc.). In other cases, it's not needed, but can be added to improve readability.

Here, it _can_ be removed for all fields excluding `event_type`, since they don't alter the readability by a great extent. It's not a necessary change, I'll leave it upto you to take a call on this.